web/frontend/src/components/slam/index.svelte

```diff
@@ -254,9 +254,11 @@
 
       try {
         hasActiveSession = true;
-        sessionId = await overrides.startMappingSession(mapName);
-        mappingSessionStarted = true;
```

Comment:
could we just use hasActiveSession here rather than tracking two different variables?

Reply:
missed that these variables already existed, sorry!